src/main/java/org/candlepin/subscriptions/inventory/db/model/InventoryHost.java

```diff
@@ -83,6 +83,7 @@
         "h.facts->'rhsm'->>'GUEST_ID' as guest_id, " +
         "h.facts->'rhsm'->>'SYNC_TIMESTAMP' as sync_timestamp, " +
         "h.facts->'rhsm'->>'SYSPURPOSE_ROLE' as syspurpose_role, " +
+        "h.facts->'rhsm'->>'SYSPURPOSE_SLA' as syspurpose_sla, " +
```

Comment:
You also need to add 'syspurpose_sla' to the @SqlResultSetMapping definition above this code. Sadly this is only detectable when the query is actually run against the HBI DB. It blows up if you deploy it against an actual database.

Reply:
Done.